ask-sdk-local-debug/src/com/amazon/ask/localdebug/lambdaContext/DebugLambdaContext.java

```diff
@@ -60,6 +60,16 @@ public String getFunctionName() {
     /**
      * Defaults to returning null cognito identity.
      */
+    @Override
```

Comment:
Can you please move the doc string to `getIdentity` function below?

Reply:
Also add doc string for this new function.

Reply:
Updated